browser/brave_browser_process_impl.cc

```diff
@@ -108,6 +113,10 @@ BraveBrowserProcessImpl::BraveBrowserProcessImpl(StartupData* startup_data)
   g_browser_process = this;
   g_brave_browser_process = this;
 
+  // initialize random seeds for farbling
+  session_token_ = base::RandUint64();
```

Comment:
we should probably move this logic into a farbling utility function or something or maybe a farbling token singleton?

Reply:
mainly I think BraveBrowserProcessImpl shouldn't have to know how to create these tokens, that logic should be encapsulated somewhere and a singleton seems like a good option here instead of putting it in browser process

Reply:
moved in https://github.com/brave/brave-core/pull/12234/commits/489b9fe4d58e62ceae62a4978b9d5d93ffcd8a3c